include/z64save.h

```diff
@@ -5,10 +5,16 @@
 #include "z64math.h"
 
 typedef enum {
+#if OOT_NTSC
+    /* 0 */ LANGUAGE_JPN,
+    /* 1 */ LANGUAGE_ENG,
+    /* 2 */ LANGUAGE_MAX
+#else
     /* 0 */ LANGUAGE_ENG,
     /* 1 */ LANGUAGE_GER,
     /* 2 */ LANGUAGE_FRA,
     /* 3 */ LANGUAGE_MAX
+#endif
```

Comment:
I think the `LANGUAGE_MAX` shouldn't be ifdef'd. Not sure what to do with the comment tho, maybe leave it blank?

Maybe something like this?

```suggestion
typedef enum {
#if OOT_NTSC
    /* 0 */ LANGUAGE_JPN,
    /* 1 */ LANGUAGE_ENG,
#else
    /* 0 */ LANGUAGE_ENG,
    /* 1 */ LANGUAGE_GER,
    /* 2 */ LANGUAGE_FRA,
#endif
    /*   */ LANGUAGE_MAX
```

Reply:
I don't mind either duplicate _max or outside _max

For outside, could also just not put a comment at all

Reply:
I played with it a bit and settled on blank comment